.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "pytorch_xla",
+  "image": "us-central1-docker.pkg.dev/tpu-pytorch/docker/development:tpu",
+  "runArgs": [
+    "--privileged"
+  ],
+  "forwardPorts": [
+    4200
```

Comment:
What is this forwarded port for?

Reply:
Forwarding the question to @stgpetrovic.

Reply:
Goran said it's sth for angular. I don't think it's needed, let's remove.